hardhat.config.ts

```diff
@@ -1,8 +1,9 @@
 import 'dotenv/config';
 import {HardhatUserConfig} from 'hardhat/types';
 import 'hardhat-deploy';
-import 'hardhat-deploy-ethers';
+import '@nomiclabs/hardhat-ethers'; // aliased to hardhat-deploy-ethers
```

Comment:
Why is this needed?

Reply:
It is because `hardhat-upgrades` depends on `hardhat-ethers` but we use a fork of it : `hardhat-deploy-ethers`
nomiclabs recentrly allowed for plugins to extends their own plugin, so `hardhat-deploy-ethers` could become such plugin instead of a fork